index.md

```diff
@@ -30,7 +30,7 @@ ms.date: "09/30/2019"
                 </a>
             </li>
             <li>
-                <a href="/dotnet/api/?view=netcore-3.0">
+                <a href="/dotnet/api/?view=netcore-3.1">
```

Comment:
I purposely removed this change from another PR. We're blocked from getting the APIs updated now.